ui-admin/src/study/participants/JustifyAndSaveModal.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
```

Comment:
Hmm, I see what you mean...`JustifyChangesModal`? I could see this modal being reused for justifying changes to other participant data, if we implement that in the future. `EditConfirmationModal` seems a bit ambiguous to me, since there is nothing that you would be editing.